salesagility/SuiteCRM
# fix extra heading space

## Description
Fix unnecessary empty space in heading in different resolution.




## Motivation and Context


## How To Test This
- go to a module and try to expand/contract the browser
![before](http://pix.toile-libre.org/upload/original/1515411092.png "before")
![after](http://pix.toile-libre.org/upload/original/1515411125.png "after")
- see if there is no huge space in heading.


## Types of changes


### Final checklist
